fh-messaging/lib/mongo_functions/metrics_map_apprequestdest.js

```diff
@@ -0,0 +1,15 @@
+//ts and emit are used for testing purposes
+var ts;
+var emit;
+function metrics_m() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Rename this 'metrics_m' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS100)